python/lsst/ts/Dome/mock_llc_statuses/amcs_status.py

```diff
@@ -26,7 +25,7 @@ def __init__(self):
         self.vmax = self.amcs_limits.vmax
         # variables helping with the state of the mock AZ motion
         self.motion_velocity = self.vmax
-        self.motion_direction = motion_dir.CW.value
+        self.crawl_velocity = 0
```

Comment:
Please add code comments explaining what `motion_velocity` and `crawl_velocity` are. I am guessing that motion_velocity is the speed during the move and crawl_velocity is the speed after the move. But if so, it really surprises me to initialize motion_velocity to self.vmax instead of 0.

Reply:
Done